recipes-extended/ofdpa-tools/ofdpa-tools/switch_tcpdump

```diff
@@ -49,6 +49,49 @@ verbose output for ARP packets arriving on port3, use:
 switch_tcpdump --inPort port3 -v 'arp'
 """
 
+MANUAL = """
```

Comment:
> BISDN Linux does not have man pages

```
accton-as4610-54:~$ find /usr/share/man | wc -l
2514
```

???

Reply:
I stand corrected. I added a man page for switch_tcpdump and removed the "--manual" option from the script.